### PR TITLE
fix overheal and indent

### PR DESCRIPTION
**Pull request by WoniMK7**
before: https://cdn.discordapp.com/attachments/257945014124937217/701023211134910514/20200418125400_1.jpg

after: https://cdn.discordapp.com/attachments/257945014124937217/701030326322855956/20200418132252_2.jpg

### CODE REVIEW

**Comment by WoniMK7** [comment]
Some details: basically, the big health customization didn't change the overheal cross size.

File: #customization/bh_size_health.res [hunk 1]
```diff
@@ -18,4 +18,13 @@
 	{
 		"font"														"bh_Font60"
 	}
-}
\ No newline at end of file
+	
+	"PlayerStatusHealthBonusImage"
+	{
+		"xpos"														"98"
+		"ypos"														"49"
```
Comment (rbjaxter) [inline #customization/bh_size_health.res:25]: this appears to be just a bit off-center; xpos 100 ypos 51 should make it a bit closer to center. you can make the changes or I can just approve this one and make them on my end, whichever works

Comment (WoniMK7) [reply #customization/bh_size_health.res:25]: How did you get this value? To get mine, I translated by (delta_font_size)/2.

Comment (rbjaxter) [reply #customization/bh_size_health.res]: If I'm being honest, I just eye-balled it hahaha. more often than not in TF2 I've had issues trying to get exact values through logical means, especially with the bonus image. when you factor in the shadow as well, it just becomes easier to eyeball in my experiences (which I hate, but meh)

**Pull request merged**